src/test/java/uk/gov/hmcts/reform/jps/services/FeeServiceTest.java

```diff
@@ -75,6 +79,36 @@ void shouldThrowIllegalArgumentExceptionWhenIdsMissing() {
             .hasMessageContaining("Fee ids missing");
     }
 
+    @Test
+    void shouldReturnFeeWhenRecordPresent() {
+        long id = 100L;
+        when(feeRepository.findByHmctsServiceIdAndJudgeRoleIdAndEffectiveFromIsLessThanEqual(
+            anyString(),
+            anyString(),
+            any(LocalDate.class)
+        )).thenReturn(Optional.of(uk.gov.hmcts.reform.jps.domain.Fee.builder()
+                                              .id(id)
+                                              .build()));
+        uk.gov.hmcts.reform.jps.domain.Fee fees = feeService.findByHmctsServiceIdAndJudgeRoleTypeIdAndSittingDate(
+            "BBA3",
+            "Judge",
+            LocalDate.now()
+        );
+
+        assertThat(fees.getId()).isEqualTo(id);
+    }
+
+    @Test
+    void shouldThrowIllegalArgumentExceptionWhenFeeMissing() {
+        LocalDate now = LocalDate.now();
```

Comment:
No need for local var here, use LocalDate.now directly

Reply:
Could have used LocalDate.now directly but sonar complains with code smell stating only invocation should be code under test in the assert block.